static/js/containers/ChannelPage.js

```diff
@@ -144,17 +143,14 @@ class ChannelPage extends React.Component<*, void> {
         <div>
           <DocumentTitle title={formatTitle(channel.title)} />
           <ChannelBreadcrumbs channel={channel} />
-          <Card
-            title={
-              <div className="post-list-title">
-                <div>{channel.title}</div>
-                <PostSortPicker
-                  updateSortParam={updatePostSortParam(this.props)}
-                  value={qs.parse(search).sort || POSTS_SORT_HOT}
-                />
-              </div>
-            }
-          >
+          <div>
```

Comment:
Would this be a good place to use `React.Fragment` instead?

Reply:
yeah, good idea! The endless battle against extra `<div>` tags